spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -203,7 +208,11 @@ ArmeriaServerConfigurator actuatorServerConfigurator(
             } else {
                 cors = null;
             }
-
+            final List<Integer> exposedPorts = Stream.of(Optional.ofNullable(managementPort),
+                                                         Optional.ofNullable(internalServicePort))
+                                                     .filter(Optional::isPresent)
+                                                     .map(Optional::get)
+                                                     .collect(toImmutableList());
```

Comment:
Could be just?
```suggestion
            final List<Integer> exposedPorts = Stream.of(managementPort, internalServicePort)
                                                     .filter(Objects::nonNull)
                                                     .collect(toImmutableList());
```

Reply:
Yes. That's right. Thanks for the suggestion.  😄